src/main/java/fruit/shop/impl/ActivityStrategyImpl.java

```diff
@@ -0,0 +1,50 @@
+package fruit.shop.impl;

```

Comment:
the `impl` package must be in the package where the interfaces are
for example: service/imp, strategy/impl, db/imp